frontend/views/containers/proposals/Mincome.vue

```diff
@@ -67,53 +40,52 @@ export default {
   ],
   data () {
     return {
-      currencies,
-      currentStep: 0,
       v: { type: Object },
       form: {
-        incomeProvided: null,
-        incomeCurrency: 'USD', // TODO: grab this as a constant from currencies.js,
-        changeReason: null
+        incomeProvided: null
       },
       ephemeral: {
         errorMsg: null,
-        // this determines whether or not to render proxy components for tests
-        dev: process.env.NODE_ENV === 'development'
+        currentStep: 0
```

Comment:
Thanks for moving this, but `v` is still up there on line 43. Like in the other file, it doesn't seem to be used. I'm guessing it was a mistake and should be removed.

Reply:
(Double-check me on that though! Maybe it's used in some other place via and event or something? Anyway... should be removed it not used anywhere, and if it is used somewhere it should be inside of `ephemeral`

Reply:
I did and it seems it's not used anywhere. removed